blocks/basic/include/gnuradio-4.0/basic/common_blocks.hpp

```diff
@@ -59,7 +59,7 @@ ENABLE_REFLECTION_FOR_TEMPLATE(builtin_counter, in, out);
 //  - use Block::set_name instead of returning an empty name
 // TODO: Inherit from Block class when create new block.
 template<typename T>
-class multi_adder : public gr::BlockModel {
+class multi_adder : public gr::lifecycle::StateMachine<multi_adder<T>>, public gr::BlockModel {
```

Comment:
In the next iteration I will rewrite this Block to inherit directly from 'Block<T>'